docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -127,6 +127,10 @@
 * __Cohort MAF__:
     * A downloaded file may be corrupted if the server data processing is terminated after 5 minutes in order to conserve server resources.
     There will be a red banner above the MAF controls to indicate the termination.
+* __ProteinPaint__:
+    * The ProteinPaint uses 1 year = 365 days, instead of 365.25 days, causing different age values to be displayed in ProteinPaint and Case Summary page.
```

Comment:
Maybe something like:

```
The ProteinPaint tool uses 1 year = 365 days, instead of 365.25 days, causing different age values to be displayed in ProteinPaint and other areas of the portal.
```
(suggestion is due to several other areas of the portal using 365.25, not just case summary page)